core/tests/webdriverio_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -178,9 +178,9 @@ describe('Enable correctness feedback and set correctness', function() {
       // Set correctness in response editor.
       responseEditor = await explorationEditorMainTab.getResponseEditor(
         'default');
-      await responseEditor.markAsCorrect();
       await responseEditor.setFeedback(await forms.toRichText('Correct!'));
       await responseEditor.setDestination('End', true, true);
+      await responseEditor.markAsCorrect();
```

Comment:
Note: in the new pipeline, an answer group cannot be marked as correct until a destination (that is not try again) has been set.